test/benchmark.js

```diff
@@ -70,14 +63,26 @@ suite
   .add('rndm', () => {
     rndm(21)
   })
-  .on('cycle', event => {
-    let name = event.target.name.padEnd('async secure-random-string'.length)
-    let hz = formatNumber(event.target.hz.toFixed(0)).padStart(10)
-    if (event.target.name === 'uid') {
-      name = '\nNon-secure:\n' + name
+
+const longestNameLength = bench.tasks.reduce((maxTaskNameLength, task) => {
+  task.addEventListener('cycle', () => {
```

Comment:
Why do we call `task.addEventListener('cycle'` inside the cycle?

Why not after the `bench.tasks.reduce`?

Reply:
You're right, there was no reason to do it that way. I will make a pull request with the changes without hardcoding.